core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -63,7 +63,9 @@ contract Liquidatable is PricelessPositionManager {
 
         // Person who is disputing a liquidation
         address disputer;
-        // Time when dispute is initiated, needed to get price from Oracle
+        // Time when liquidation is initiated, needed to get price from Oracle
+        uint liquidationTime;
+        // Time when the dispute is initialized
```

Comment:
Why can't we get rid of the `disputeTime` variable?

Reply:
i had the same comment, I believe we can and should before we merge this

Reply:
ahh, I should've read your comments before I added mine :)

Reply:
am happy to get rid of it if we are sure there is no need for it on chain. We emit an event on dispute so can always see when the position was disputed anyways.

Reply:
have removed this.

Reply:
Sweet, thanks.